src/social-media-post/models/social-media-post.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { UserEntity } from '../../user/models/user.entity';
+
+@Entity('social_posts')
+export class SocialMediaPostEntity {
```

Comment:
Maybe an expiration date might be suitable here? It's for two given examples.
1. Posting on FaceBook about missing B+ blood in Pomeranian Voyvodenship. That might be from today for the upcoming 3 days.
2. Posting on FaceBook on absolutely FENONEMAL app called BloodDonor, that everyone can download from AppStore, which will be viable from January 1st, 1970 till infinity from today.

Reply:
1. I'm not really well versed in social media so how would that work in the context of FB (or any for that matter) post? Where would the expiration date be visible? When you make a post is there a field for it? After all the date can be posted in the contents of the post.

Reply:
I assume that by an expiration date you mean when do we stop showing that post? :) Sounds good to me, but would require additional work to run a cronjob that would expire posts after a set amount of time. In case a post gets viral though, and gets many visits from our app, do we really want to expire it?

Reply:
It's more of an info, when user will not have the possibility to add the post. Some automatically generated posts might have 3 days of "informing the user, that this post can be... uhh... posted?" and some might be shown to the user for a longer period of time

Reply:
Okay, deleted_at works for my point here.